site/src/pages/TaskPage/TaskPage.stories.tsx

```diff
@@ -202,38 +209,40 @@ export const SidebarAppHealthy: Story = {
 	beforeEach: () => {
 		spyOn(data, "fetchTask").mockResolvedValue({
 			prompt: "Create competitors page",
-			workspace: {
-				...MockWorkspace,
-				latest_build: {
-					...MockWorkspace.latest_build,
-					has_ai_task: true,
-					ai_task_sidebar_app_id: "claude-code",
-					resources: mockResources({
-						claudeCodeAppOverrides: {
-							health: "healthy",
-						},
-					}),
+			workspace: mockTaskWorkspace(
+				{
+					...MockClaudeCodeApp,
+					health: "healthy",
 				},
-			},
+				MockVSCodeApp,
+			),
 		});
 	},
 };
 
-const mainAppHealthStory = (health: WorkspaceApp["health"]) => ({
+export const SidebarAppUnhealthy: Story = {
 	beforeEach: () => {
 		spyOn(data, "fetchTask").mockResolvedValue({
 			prompt: "Create competitors page",
-			workspace: {
-				...MockWorkspace,
-				latest_build: {
-					...MockWorkspace.latest_build,
-					resources: mockResources({
-						claudeCodeAppOverrides: {
-							health,
-						},
-					}),
+			workspace: mockTaskWorkspace(
+				{
+					...MockClaudeCodeApp,
+					health: "unhealthy",
```

Comment:
This story does not show up any different in the UI to the 'healthy' one above. Is that intentional?

Reply:
Now it does 🙏